lib/go-tc/serverchecks.go

```diff
@@ -108,37 +107,53 @@ type Servercheck struct {
 }
 
 // A Single Servercheck struct for Update and Create to depict what changed
-// swagger:model ServercheckPost
 type ServercheckPost struct {
 
 	// The Servercheck data to submit
 	//
 	// Name of the server check type
 	//
 	// required: true
-	Name string `json:"servercheck_short_name" db:"servercheck_short_name"`
+	Name string `json:"servercheck_short_name"`
 
 	// ID of the server
 	//
-	// required: true
-	ID int `json:"id" db:"id"`
+	ID int `json:"id"`
 
 	// Name of the server
-	HostName string `json:"name" db:"name"`
```

Comment:
let me put them back - I was not using sqlx for the sql queries but probably better to have them on the model

Reply:
Ahh the reason why I didnt was because each one of these fields map to different fields in different tables and even that value maps to a different column name in the servercheck table based on the servercheck_short_name